src/sql/src/plan.rs

```diff
@@ -550,6 +549,23 @@ pub enum ReplicaConfig {
     },
 }
 
+#[derive(Clone, Debug, Deserialize, Serialize, PartialOrd, PartialEq, Eq, Ord)]
+pub enum ClusterSchedule {
+    /// The system won't automatically turn the cluster On or Off.
+    Manual,
+    /// The cluster will be On when a REFRESH materialized view on it needs to refresh.
+    /// The `Duration` is the `REHYDRATION TIME ESTIMATE`, i.e., how much time before a refresh to
+    /// turn the cluster On, so that it can rehydrate already before the refresh time.
+    Refresh(Duration),
```

Comment:
I think the purpose of the `Duration` here is non-obvious enough that it deserves a struct variant.

```suggestion
    Refresh {
        rehydration_time_estimate: Optional<Duration>,
    }
```

Reply:
Ok, done